database/users_models.py

```diff
@@ -236,7 +245,9 @@ def __repr__(self):
 for i in range(12):
     list_study_env.append(study_env["Study_env_preferences"][f"{i+1}"])
     list_study_time.append(
-        study_time["Study_time_preferences"][f"{i+1}"]["Time"] + " " + study_time["Study_time_preferences"][f"{i+1}"]["Frame"]
+        study_time["Study_time_preferences"][f"{i+1}"]["Time"]
+        + " "
+        + study_time["Study_time_preferences"][f"{i+1}"]["Frame"]
```

Comment:
let's use f-string here:

```suggestion
        f"{study_time["Study_time_preferences"][f"{i+1}"]["Time"]} {study_time["Study_time_preferences"][f"{i+1}"]["Frame"]}"
```